tests/Handler/Mapping/ClassName/SuffixTest.php

```diff
@@ -12,12 +12,19 @@ class SuffixTest extends TestCase
     /** @dataProvider examples */
     public function testCanAddSuffixToClassName(string $suffix, string $command, string $expectedResult) : void
     {
-        self::assertEquals(
+        self::assertSame(
```

Comment:
No complaints but curious why the change, just general strictness? :smile_cat:

Reply:
Strictness is safely :)

Reply:
Or the illusion of safety sometimes, but duly noted :+1: